packages/origin-ui-core/src/components/AutoSupplyDeviceTable.tsx

```diff
@@ -60,7 +60,7 @@ export function AutoSupplyDeviceTable() {
                 parseInt(requestedFilters[1]?.selectedValue, 10) || 0
             );
         } else {
-            entities = await deviceClient.getMyDevice();
+            entities = await deviceClient.getMyDevices(true);
```

Comment:
I think we don't need SM reads here.

Reply:
They are used in that screen:

![Screenshot 2020-06-15 at 10 12 00](https://user-images.githubusercontent.com/9353642/84633451-b200c380-aef0-11ea-8e27-740e15ca5650.png)